deploy/monitor/prometheus_rule.yaml

```diff
@@ -0,0 +1,35 @@
+# Monitor Service (Metrics)
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    monitoring-key: middleware
+    prometheus: application-monitoring
+    role: alert-rules
+  name: application-monitoring
+spec:
+  selector:
+    matchLabels:
+      name: unifiedpush-operator
+  groups:
+    - name: general.rules
+      rules:
+      - alert: UnifiedpushOperatorDown
+        expr: absent(up{job="unifiedpush-operator"} == 1)
+        for: 5m
+        labels:
+          severity: critical
+        annotations:
+          description: "The unifiedpush-operator has been down for more than 5 minutes. "
+          summary: "The unifiedpush-operator is down. For more information see on the MSS operator https://github.com/aerogear/unifiedpush-operator"
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.2.0/SOP/SOP-operator.adoc"
```

Comment:
Can we leave this as a TODO until we have the versions and files in place.

Reply:
👍